Formula/elvish.rb

```diff
@@ -16,10 +16,12 @@ class Elvish < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    version = `git describe --tags --always`.strip
+    goroot = `go env GOROOT`.strip
     (buildpath/"src/github.com/elves/elvish").install buildpath.children
     cd "src/github.com/elves/elvish" do
       system "go", "build", "-ldflags",
-             "-X github.com/elves/elvish/build.Version=#{version}", "-o",
+             "-X github.com/elves/elvish/buildinfo.Version=#{version} -X github.com/elves/elvish/buildinfo.GoPath=#{buildpath} -X github.com/elves/elvish/buildinfo.GoRoot=#{goroot}", "-o",
              bin/"elvish"
```

Comment:
The current stable version already reports its version correctly so this must be a head specific change. We could scope it to head, but it's probably worth just ignoring until the next stable release.

Reply:
Using #{version} results in a different version string than Elvish generates with other build methods (at least on HEAD, and eventually the next stable release).

Reply:
That's not actually a bug just a behavior.

Reply:
So just to clarify, would this be something to ignore for now, but bring back up for the next stable release?

For some more info, pre-built binaries hosted on dl.elv.sh already have this info filled out as part of their build process.  Unfortunately, with stable releases few and far between most Elvish users just use HEAD, and update regularly.  Having different versioning schemes between different OSes, or even on the same OS with different installation methods (`go get` vs `brew install`) seemed like it could be confusing.